app/domain/logs.rb

```diff
@@ -4,7 +4,7 @@ module LogMessages
 
   module Conjur
 
-    PrimarySchema = Util::TrackableErrorClass.new(
+    PrimarySchema = ::Util::TrackableLogMessageClass.new(
```

Comment:
why was :: added? From this commit, it doesn't seem that there was a change in directory

Reply:
i rooted the module so there is no confusion with other Util modules